### PR TITLE
188 About page regression fix

### PR DESCRIPTION
**Pull request by Alexandre-Georges**
## Reminder

> Please ensure the following criteria is met for this PR. It will help others review your PR and provide confidence things work as expected.

- [x] CI is green
- [x] Link to any related issues
- [x] Received at least 1 approval from core members
- [x] Made sure that changes are obvious for the reviewer (ex meaningful title and commit messages, comments in code or PR where appropriate, screenshots, etc.)

# What Changed?

> Related Issue: [Link](https://github.com/commitdev/commit.dev/issues/188)

The previous implementation broke the about page as the index and about page used the same component. The issue was that the component is not reusable because the layout and styling are different on the 2 pages.

I refactored that part to have 2 independent sections.

## Screenshots
Screen Shot 2021-04-09 at 9 34 09 AM
Screen Shot 2021-04-09 at 9 34 22 AM

### CODE REVIEW

File: components/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export { default as HighlightedText } from './highlighted-text'
 export { default as Input } from './input'
 export { default as Logo } from './logo'
 export { default as Page } from './page'
+export { default as RowContainer } from './row-container'
```
Comment (Alexandre-Georges) [inline components/index.js:12]: The only reusable piece is the container for images. This is a simple flex box container.

File: pages/about.js [hunk 2]
```diff
@@ -28,10 +29,7 @@ const About = () => (
       </div>
       <DiversityAndInclusionSection />
       <ExecutiveTeamSection />
-      <LogoSection
```
Comment (Alexandre-Georges) [inline pages/about.js:31]: The index page and the about page shared the same component (`LogoSection`). But the layout is significantly different making this shared component really hard to reuse.

I chose to have 2 sections: one for the index and one for the about page. The only shared piece is the image container.

**Review by alebiavati — APPROVED** [review]

File: components/row-container/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+const RowContainer = ({ className, children }) => (
+  <Container className={className}>{children}</Container>
+)
+
+RowContainer.propTypes = {
+  className: string,
+  children: arrayOf(element),
+}
+
+export default RowContainer
```
Comment (alebiavati) [inline components/row-container/index.js]: non-blocker: this could be just:
```
export const RowContainer = styled.div`
  ...
`
```

Comment (Alexandre-Georges) [reply components/row-container/index.js]: Right, addressed!

**Pull request merged**